lomap/graphgen.py

```diff
@@ -901,15 +963,23 @@ def max_dist_mol(mol):
             for each_node in self.resultGraph:
                 
                 id_mol = self.resultGraph.node[each_node]['ID']
-                mol = AllChem.RemoveHs(self.dbase[id_mol].getMolecule())
+                #sliu 07/17 skip remove Hs by rdkit if Hs cannot be removed
+                try:
+                    mol = AllChem.RemoveHs(self.dbase[id_mol].getMolecule())
+                except:
+                    mol = self.dbase[id_mol].getMolecule()
             
                 # max_dist = max_dist_mol(mol)
                 # if max_dist > 7.0:
                 #     continue
 
                 AllChem.Compute2DCoords(mol)
-                
-                img_mol = Draw.MolToImage(mol,mol_size)
+                #sliu 07/17 add try exception for cases cannot be draw
```

Comment:
Cannot remember if there is any cases catch in the try exception. Maybe this try exception is just for debugging and also let the code bypass the Draw function instead of breaking the for loop. Add more words in the exception log though.